conf.yaml

```diff
@@ -905,6 +905,25 @@ contents:
 
     -   title:      "Observability: APM, Logs, Metrics, and Uptime"
         sections:
+          - title:      Observability
+            prefix:     en/observability
+            current:    *stackcurrent
```

Comment:
Depending on when you merge this, you'll need to make sure there's a 7.8 branch if you use `stackcurrent`. Other option is to set this to master.

Reply:
Yeah, we need `master` here